deploy/scripts/src/cri-kiwi/BAV.ts

```diff
@@ -9,6 +9,7 @@ import { isStatusCode200, pageContentCheck } from '../common/utils/checks/assert
 import { sleepBetween } from '../common/utils/sleep/sleepBetween'
 import { bankingPayload } from './data/BAVdata'
 import { getAuthorizeauthorizeLocation, getClientID, getCodeFromUrl, getAccessToken } from './utils/authorization'
+import { getEnv } from '../common/utils/config/environment-variables'
```

Comment:
Same as previous comment - I will include this change in `[PR-544](https://github.com/govuk-one-login/performance-testing/pull/544)` (Kiwi scripts still pending). 
We can keep the scope of this PR to utility function documentation and merge this change via the other PR if that's okay.